src/components/input/input.scss

```diff
@@ -210,22 +210,46 @@ md-input-container {
     @include rtl(padding-left, 0, $input-container-padding);
   }
 
-  //
-  // ngMessage base styles - animations moved to input.js
-  //
+  // ngMessage base styles
   .md-input-messages-animation {
     position: relative;
     order: 4;
     overflow: hidden;
     @include rtl(clear, left, right);
+  }
 
-    &.ng-enter {
-      // Upon entering the DOM, messages should be hidden
-      .md-input-message-animation {
-        opacity: 0;
-        margin-top: -100px;
-      }
+  .md-input-message-animation {
+    // Enter animation
+    // Pre-animation state is transparent and off target
```

Comment:
What exactly do you mean by "off target" here? Can you elaborate or use a different terminology?